### DISCUSSION TITLE
[Question][Hw3] builder_before issue

### DISCUSSION THREAD

**Discussion by Quasar0311** [prosyslab-classroom/cs348-information-security#537] [Q&A]
According to documentation, builder_before is used like the below figure.

![image](https://user-images.githubusercontent.com/52192541/116821963-42063500-abb7-11eb-9f37-047c860bd356.png)

it seems to need two parameters llcontext and llvalue, but description says 'builder_before ins' which using only one parameter. What is the syntax for using this function?

**Comment by Junseon00** [comment]
Yes, it has two arguments llcontext and llvalue.

**Comment by Quasar0311** [comment]
Thank you!
